galasa-ui/src/tests/components/UsersTable.test.tsx

```diff
@@ -43,19 +43,19 @@ jest.mock('next-intl', () => ({
     const translations: Record<string, string> = {
       "searchPlaceholder": "Search",
       "ariaLabel": "Users table",
-      "headers_loginId": "Login ID",
-      "headers_role": "Role",
-      "headers_lastLogin": "Last login",
-      "headers_lastAccessTokenUse": "Last access token use",
-      "headers_actions": "Actions",
+      "headersLoginId": "Login ID",
+      "headersRole": "Role",
+      "headersLastLogin": "Last login",
+      "headersLastAccessTokenUse": "Last access token use",
+      "headersActions": "Actions",
       "editIconDescription": "Edit User",
       "deleteIconDescription": "Delete User",
-      "modal_label": "Delete User",
-      "modal_heading": "Are you sure you want to delete '{user}'?",
-      "modal_primaryButton": "Delete",
-      "modal_secondaryButton": "Cancel",
-      "modal_notificationTitle": "Deleting a user will remove any memory the Galasa service has of this user.",
-      "modal_notificationSubtitle": "Any access tokens previously allocated by this user will be removed and will no longer have access to the Galasa service.\n\nTests run by this user will remain untouched and available for queries.\n\nIf the user logs in to the Galasa service after this point, they will be challenged for credentials from the configured authorization authority, as they were originally when they started using the Galasa service. If that authority allows access, a new set of details for that user will be collected from that point.",
+      "modalLabel": "Delete User",
+      "modalHeading": "Are you sure you want to delete '{user}'?",
+      "modalPrimaryButton": "Delete",
+      "modalSecondaryButton": "Cancel",
+      "modalNotificationTitle": "Deleting a user will remove any memory the Galasa service has of this user.",
+      "modalNotificationSubtitle": "Any access tokens previously allocated by this user will be removed and will no longer have access to the Galasa service.\n\nTests run by this user will remain untouched and available for queries.\n\nIf the user logs in to the Galasa service after this point, they will be challenged for credentials from the configured authorization authority, as they were originally when they started using the Galasa service. If that authority allows access, a new set of details for that user will be collected from that point.",
```

Comment:
Seems a bit bad we have to re-define these to be exactly the same in the test bundle. Why do this ?